src/app/main/main.component.js

```diff
@@ -40,6 +49,21 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       url: "/checkout.html",
       template: '<checkout></checkout>'
     })
+    .state('profile', {
+      url: "/profile",
+      abstract: true,
```

Comment:
Thanks for letting me know @Omicron7 , I will look at your examples and take care of my PR